java/Cherepnin_Oleh/src/com/zoolatech/lecture3/tasks/_2/NameValidator.java

```diff
@@ -0,0 +1,30 @@
+package com.zoolatech.lecture3.tasks._2;
+
+import java.lang.reflect.Field;
+
+public class NameValidator implements SmallerValidator {
+
+    private String regex = "[A-Z][a-z]+";
+
+    @Override
+    public boolean isValid(User user) {
+        try {
+            boolean firstNameCheck = printMessage(user.getClass().getDeclaredField("firstName"), user.getFirstName());
```

Comment:
```
user.getClass().getDeclaredField("firstName")
```

You've already defined a field name as a string literal. Why do you need to retrieve this field through reflection and get the same value?